lib/config/index.js

```diff
@@ -1712,7 +1713,14 @@ function getOrCreateInstance() {
 
       // Config construction has potential to throw due to invalid settings.
       // This allows the agent to return a stub api without crashing the process.
-      _configInstance = Object.assign(defaultConfig(), { agent_enabled: false })
+      _configInstance = Object.assign(defaultConfig(), {
+        agent_enabled: false,
+        logging: {
+          enabled: true,
+          filepath: process.stdout
```

Comment:
ok i finally figured out a way to test this and I'm glad I did.  this should be `stdout` not process.stdout.  process.stdout is already a WriteStream so it'll fail when trying to create the writeStream in lib/logger with

```sh
node:internal/fs/utils:670
    throw new ERR_INVALID_ARG_TYPE(propName, ['string', 'Buffer', 'URL'], path);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received an instance of WriteStream
    at new WriteStream (node:internal/fs/streams:325:5)
    at Object.createWriteStream (node:fs:2889:10)
    at Object.<anonymous> (/Users/revans/code/node-newrelic/lib/logger.js:50:21)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/revans/code/node-newrelic/index.js:17:14) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

I also realized we lack tests for lib/logger but i'll take care of that in a different PR.

Reply:
A way to manually test this is to add these 2 lines in a config

```
  high_security: true,
  security_policies_token: 'ffff',
```

I found that out by searching in the `test/unit/config` for t.throws and it found 

```js
tap.test('should throw with both high_security and security_policies_token defined', (t) => {
  t.throws(function testInitialize() {
    Config.initialize({
      high_security: true,
      security_policies_token: 'fffff'
    })
  })

  t.end()
})
```

Reply:
I'm glad you tested it too!  Great catch.  I should've added a test case for that fallback config to begin with.  I made the change and added a quick test case that proves that the logger does get enabled and piping to stdout when the config is invalid.

Reply:
wow, didn't realize we had those tests.  I'm still new-ish on the team.  thanks for adding that test 🙏🏻